src/components/pages/Bookmarks.js

```diff
@@ -1,11 +1,104 @@
 import Header from '../modules/Header'
+import GetLists from '../services/LocalStorage/GetList'
+import styled from 'styled-components/macro'
+import { useState, useEffect } from 'react'
+import '../modules/page_components/ProfileCSS.css'
 
-const Booking = () => {
+//
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        width: '100%',
+    },
+    heading: {
+        fontSize: theme.typography.pxToRem(15),
+        fontWeight: theme.typography.fontWeightRegular,
+    },
+}));
+//
+
+export default function Profile() {
+    const [shopThisList, setShopThisList] = useState(GetLists('shoppingList') ?? {})
+
+    //
+    const classes = useStyles()
+    //
+
+    const ingredientsByRecipe = [];
+    console.log(shopThisList)
+    for (let recipeTitle in shopThisList) {
+        ingredientsByRecipe.push({ title: recipeTitle, shoppingList: shopThisList[recipeTitle] });
+    }
+
+
+    console.log(ingredientsByRecipe)
     return (
         <div>
             <Header title="Bookmarks" />
+            <ProfileWrapper>
+                <div className={classes.root}>
+                    {ingredientsByRecipe.map(ingredientList => (
+                        <Accordion>
+                            <AccordionSummary>
+                                <Typography>
+                                    <ul className='ULement'>{ingredientList.title}</ul>
+                                </Typography>
+                                <Button className='Button' onclick={() => { alert('Zomg, i´am clicked') }}>Delete</Button>
```

Comment:
In react you'll need to write onClick with a capital c

Reply:
Thank you, damn typos... 🙂